x/gov/simulation/genesis_test.go

```diff
@@ -40,7 +40,7 @@ func TestRandomizedGenState(t *testing.T) {
 	var govGenesis types.GenesisState
 	simState.Cdc.MustUnmarshalJSON(simState.GenState[types.ModuleName], &govGenesis)
 
-	dec1, _ := sdk.NewDecFromStr("0.457000000000000000")
+	dec1, _ := sdk.NewDecFromStr("0.466000000000000000")
```

Comment:
just curious, how did this / why did this fix simulator?

Reply:
so this didn't fix sim, instead this https://github.com/osmosis-labs/cosmos-sdk/pull/457/files#diff-699f0ba2abfc2c68ec29cf1588cb8630e0cb9488013dba8d5811d7851d612472R55
is what fixed sim. 

while doing so i also realized the cause of error here https://github.com/osmosis-labs/cosmos-sdk/pull/457#discussion_r1210821555